cos-fleetshard-it/cucumber/src/main/java/org/bf2/cos/fleetshard/it/cucumber/ConnectorContext.java

```diff
@@ -52,7 +51,7 @@
     private final List<ManagedConnector> history = new CopyOnWriteArrayList<>();
 
     @Inject
-    private CosFeatureContext cosCtx;
+    CosFeatureContext cosCtx;
```

Comment:
Why the change in scope?

Reply:
Quarkus complains since it has to use reflection to inject it, and from looking at other Injection points we seem to be following this pattern of package-private.